deploy/operator/monitoring-stack-operator-cluster-role.yaml

```diff
@@ -118,6 +118,16 @@ rules:
   - list
   - update
   - watch
+- apiGroups:
+  - integreatly.org
+  resources:
+  - grafanas
+  verbs:
+  - create
+  - get
```

Comment:
wondering if we should remove `get` ?

Reply:
Yes, good idea. I'll try it out.

Reply:
Let me merge this and I'll follow up with a PR to remove get